submission/persistence/src/main/kotlin/ac/uk/ebi/biostd/persistence/mapping/SubmissionDbMapper.kt

```diff
@@ -55,6 +56,7 @@ class SubmissionDbMapper {
             modificationTime = toInstant(submissionDb.releaseTime)
             releaseTime = toInstant(submissionDb.releaseTime)
             section = sectionMapper.toSection(submissionDb.rootSection)
+            processingStatus = ProcessingStatus.valueOf(submissionDb.status)
```

Comment:
database records saved and retrieve by backend is not like a web API input that user can change so we can assume that value is a proper one. Not need the validation of unexpected value IMO.